### ISSUE TITLE
Screen reader reads incorrect column number

### ISSUE THREAD

**Issue by shivanip9** [ag-grid/ag-grid#7782]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => see 'Providing a Reproducible Scenario'
[] feature request => do not use Github for feature requests, see 'Customers of AG Grid'
[] support request => see 'Requesting Community Support'
```

**Current behavior**
When navigating the ag-grid using keyboard navigation (ctrl + alt + arrow keys), the screen reader doesn't read the correct column number.
Repro steps:
1. Go to https://ag-grid.com/example/.
2. Turn on NVDA screen reader.
3. Tab through the point where the focus enters inside the table.
4. Now, navigate inside the table through the end of the first row using (ctrl + alt + right arrow key).
5. Once you reach the end, navigate back to the first column (ctrl + alt + left arrow key).
6. Observe that NVDA reads incorrect column number for many columns while navigating back.

The aria-colindex is correct but the column number announced by the screen reader doesn't always match with aria-colindex value for a column.

**Expected behavior**
Screen reader should announce the correct column number for the columns otherwise users will be confused.

* **AG Grid version:** [27.2.0]
Issue exists in the latest version as well.

* **Language:** [all]

**Comment by AG-Zoheil** [comment]
Hello 

Thank you for reporting this - much appreciated!

Based on your description we've now reproduced this issue as a bug.

We have added this bug to our backlog and we are tracking it with the following reference: **AG-11126**

We try to fix bugs from one release to the next, so this should be fixed in the next release or the one after if it was raised too close to the next release date.

See whether this item will be in the next release by checking the NEXT RELEASE checkbox on the product pipeline page:
https://www.ag-grid.com/ag-grid-pipeline/

The best way to track this is to sign up for AG Grid new release notifications using [the instructions here](https://ag-grid.zendesk.com/hc/en-us/articles/360002648058-How-do-I-sign-up-for-ag-Grid-release-notifications-). This way you'll know as soon as a new version is out and you can check whether this specific item was implemented on [the changelog page](https://www.ag-grid.com/ag-grid-changelog/).

Thanks again for bringing this up with us.

Kind regards,
Zoheil

**Comment by shivanip9** [comment]
Hi @AG-Zoheil, could you please provide an ETA for this bug fix? I searched for it in the changelog and the backlog but couldn’t locate it.

**Comment by shivanip9** [comment]
Hi @AG-Zoheil, the status of this bug was scheduled for next release until last week but today, the status is changed to Rejected. What does it mean?

<img width="899" alt="Image" src="https://github.com/user-attachments/assets/3b414f50-9a3a-4124-ab4a-192fa8afd96f" />

**Comment by AG-Zoheil** [comment]
Hi,

This is not a bug, it’s a result of Column Virtualisation. In order to have proper announcements with this type of navigation they need to disable virtualisation.

We go over this here: [JavaScript Grid: Accessibility | AG Grid](https://www.ag-grid.com/javascript-data-grid/accessibility/#ensure-all-grid-elements-are-always-rendered). Please have a look.

Kind regards,
Zoheil
